obspy/signal/spectral_estimation.py

```diff
@@ -972,10 +980,12 @@ def __process(self, tr):
             w = w[::-1]
             # Here we do the response removal
             # Do not differentiate when `special_handling="hydrophone"`
-            if self.special_handling == "hydrophone":
+            if self.special_handling == "hydrophone" or \
```

Comment:
I would rather go for `if self.special_handling in ("hydrophone", "infrasound")`

Reply:
That is better - thanks!